test/util/removeUnnecessaryParens.test.js

```diff
@@ -22,7 +22,7 @@ describe('removeUnnecessaryParens', function () {
     ['(((-5)))', '-5'],
     ['((4+5)) + ((2^3))', '(4 + 5) + 2^3'],
     ['(2x^6 + -50 x^2) - (x^4)', '2x^6 - 50x^2 - x^4'],
-    ['(x+4) - (12 + x)', 'x + 4 - (12 + x)'],
+    ['(x+4) - (12 + x)', '(x + 4) - (12 + x)'],
```

Comment:
Oh, that what was proposed by @kevinbarabash in #126. I thought you agreed on that. That is actually even better, I will change ASAP